src/spaces.rs

```diff
@@ -293,7 +295,28 @@ pub fn metrics<'a, T: ParserTrait>(parser: &'a T, path: &'a Path) -> Option<Func
         cursor.reset(node.object());
         if cursor.goto_first_child() {
             loop {
-                children.push((Node::new(cursor.node()), new_level));
+                // FIX ME: This code fixes the python parser error for comments , when the tree-sitter-python parser will be fixed revert the changes
+                // Issue on the tree-sitter-python github: https://github.com/nvim-treesitter/nvim-treesitter-textobjects/issues/25
+                // Link to PR resolving this issue : https://github.com/tree-sitter/tree-sitter-python/pull/143
+                if lang == LANG::Python && cursor.node().kind() == "function_definition" {
```

Comment:
Honestly, I don't like this patch...
I didn't dive into the detail of implementation here but the function (`metrics`) here is language agnostic and you're adding a check for a specific language (python) so we'll have this check for all the supported languages.

Reply:
Ok, i'll see if i find a way to resolve this issue by not influencing the other supported languages